### ISSUE TITLE
Examples not working

### ISSUE THREAD

**Issue by halboffen** [negomi/react-burger-menu#196]
I get the following warnings during install and errors during start:

```
> git clone https://github.com/negomi/react-burger-menu.git
> cd react-burger-menu
> npm install
```

```
npm WARN deprecated gulp-minify-css@1.2.4: Please use gulp-clean-css
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated connect@2.30.2: connect 2.x series is deprecated
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1241 packages in 33.925s
```

`> npm start`

```
> react-burger-menu@2.1.6 start /home/bru/sidebar/react-burger-menu
> gulp dev

/home/bru/sidebar/react-burger-menu/node_modules/react-component-gulp-tasks/node_modules/require-dir/index.js:93
            if (!require.extensions.hasOwnProperty(ext)) {
                                    ^

TypeError: require.extensions.hasOwnProperty is not a function
    at requireDir (/home/bru/sidebar/react-burger-menu/node_modules/react-component-gulp-tasks/node_modules/require-dir/index.js:93:37)
    at Object.<anonymous> (/home/bru/sidebar/react-burger-menu/node_modules/react-component-gulp-tasks/node_modules/gulp-git/index.js:4:18)
    at Module._compile (module.js:573:30)
    at Module._extensions..js (module.js:584:10)
    at Object.require.extensions.(anonymous function) [as .js] (/home/bru/sidebar/react-burger-menu/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-burger-menu@2.1.6 start: `gulp dev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-burger-menu@2.1.6 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/bru/.npm/_logs/2017-09-12T11_52_01_714Z-debug.log

```
Thanks for any hints
Peter

**Comment by negomi** [comment]
Hi @halboffen,

It looks like one of the dependencies is causing this, when the project is run with Node 8: https://github.com/JedWatson/react-component-gulp-tasks/pull/30

Hopefully they merge that soon, but for now, you might have to use Node 7 to avoid the error.

**Comment by sedubois** [comment]
It seems that react-component-gulp-tasks is not maintained any more. Is there any other way to fix this?

**Comment by negomi** [comment]
That project does seem abandoned, so I've switched to using a fork of it for now.

Fixed in `v2.2.1` - confirmed react-burger-menu now builds/runs locally with the latest stable Node (9.2.1 as of today).
